provisioning/osc-install

```diff
@@ -183,18 +216,14 @@ do_prep() {
   echo "Beginning action: Prep"
   yum -y install http://dl.fedoraproject.org/pub/epel/7/x86_64/e/epel-release-7-5.noarch.rpm
   sed -i -e "s/^enabled=1/enabled=0/" /etc/yum.repos.d/epel.repo
-  sed -i -e "s/^enabled=1/enabled=0/" /etc/yum.repos.d/ose.repo
   yum -y update
   yum -y --enablerepo=epel install https://kojipkgs.fedoraproject.org//packages/ansible/1.8.4/1.el7/noarch/ansible-1.8.4-1.el7.noarch.rpm
 }
 
 do_dns() {
   echo "Beginning action: DNS"
-  command="bash ${SCRIPT_BASE_DIR}/osc-dns-config -m=${MASTER} -n=${NODES} -b=${OPENSHIFT_BASE_DOMAIN} -d=${NODES%%,*}"
+  command="bash ${SCRIPT_BASE_DIR}/osc-dns-config -m=${MASTER} -n=${NODES} -b=${OPENSHIFT_BASE_DOMAIN} -d=${NODES##*,}"
```

Comment:
Would it be good to use the "find_infra_node" from above and run DNS on that same node, or does it not really make send and you'd want to just use the last node?